core/controller/cluster/controller_operation.go

```diff
@@ -272,6 +274,12 @@ func (c *controller) Rollback(ctx context.Context,
 		return nil, err
 	}
 
+	// Deprecated: for internal usage
```

Comment:
// what about as below 
// Deprecated：for internal usage
if err := checkVersionChange() {
    SynGitOpsBranch(.....)
}